src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,33 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, onTabSelected }) => {
+  const activeTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={cn({ 'is-active': selectedTabId === tab.id })}
```

Comment:
```suggestion
              className={cn({ 'is-active': activeTab.id === tab.id })}
```

to fix the tests you should rewrite your logic using `activeTab.id` instead of `selectedTabId`